<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_close">
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>18</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
   color:rgb(85, 170, 255);
	font: 75 18pt &quot;Adobe Devanagari&quot;;

}
QLabel:hover{
   background-color:rgb(255,255,255,50%);
}</string>
      </property>
      <property name="text">
       <string>×</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>451</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 75 16pt &quot;Adobe Caslon Pro Bold&quot;;
  background-color:rgb(0,0,0,0%);
color:rgb(255,255, 255)
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
      </property>
      <property name="text">
       <string>ip address            </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_ip">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>18</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
  border:1px solid rgb(219, 211, 222);
  border-radius:2px;
}
QLineEdit:focus{
  border:1px solid rgb(78, 125, 255);
  border-radius:2px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>10.60.102.252</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 75 16pt &quot;Adobe Caslon Pro Bold&quot;;
  background-color:rgb(0,0,0,0%);
color:rgb(255,255, 255)
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
      </property>
      <property name="text">
       <string>port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
  border:1px solid rgb(219, 211, 222);
  border-radius:2px;
}
QLineEdit:focus{
  border:1px solid rgb(78, 125, 255);
  border-radius:2px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>20202</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 9pt &quot;Adobe 黑体 Std R&quot;;
  background-color:rgb(85, 170, 255);
color:white;
}
QPushButton:hover{

  background-color:rgb(85, 170, 255);
}</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
